### DISCUSSION TITLE
The request address added by iframe supports environment variables

### DISCUSSION THREAD

**Discussion by willardwang** [apache/skywalking#11131] [General]
<img width="292" alt="image" src="https://github.com/apache/skywalking/assets/3304388/683e8d1c-063f-4da1-9fa6-8a9bb5028a0e">
The request address added by iframe supports environment variables
<img width="574" alt="image" src="https://github.com/apache/skywalking/assets/3304388/8deb5e8c-731b-48b3-8e79-8085e3fe2972">
Like the above server address plus IP if it is written dead we even export this content but to the production environment or need to change again

**Comment by wu-sheng** [comment]
This had been discussed but not going to be supported.
As this is not typical case, there is not point to add complexity.

**Reply by willardwang** [reply]
Well, thank you very much. I'd like to ask you again
![image](https://github.com/apache/skywalking/assets/3304388/8d859cb4-6bbe-4034-a62d-0f1048dabfd4)
Here the default display logic can be customized to change it?
I want to show by default the logical endpoints that we care about the most, we will have a prefix after these logical endpoints, and then support his fuzzy query when searching

**Reply by wu-sheng** [reply]
No fuzzy in elastic search cases.
It support word split only.

**Reply by willardwang** [reply]
Yes you are right, I am ES is participle, I tried participle is OK，Now I need to figure out how to get him to default to showing only my own logical endpoints, which is equivalent to entering the default query criteria. Can I do this?

**Reply by willardwang** [reply]
I'm going to give all my logical endpoints to do the corresponding segmentation, and then I can do the segmentation query, and then I need to open this endpoint_list page and by default only show the logical endpoints that I care about，I see he finally launched graphql can customize this?
